entrypoint.sh

```diff
@@ -9,9 +9,19 @@ echo "::add-matcher::${RUNNER_TEMP}/_github_workflow/flake8-matcher.json"
 # The weird piping is needed because we want to get the exitcode from flake8,
 # but there is no other good universal way of doing so.
 # e.g. PIPESTATUS and pipestatus only work in bash/zsh respectively.
-echo "Running flake8 on '${INPUT_PATH}' ..."
+echo "Running flake8 on '${INPUT_PATH}' with the following options..."
+command_args=""
+echo "Ignoring '${INPUT_IGNORE}'"
+if [ "x${INPUT_IGNORE}" != "x" ]; then 
```

Comment:
I always prefer: `[ -z "${INPUT_IGNORE}" ]` over the "x" method. But this is more preference than anything else :D

Reply:
I thought the x thing is slightly more cross-compatible or safer if the variable doesn't exist or something (although I guess quoting may catch that). Happy to change it if you'd prefer.